omod/src/main/resources/config.xml

```diff
@@ -74,7 +74,8 @@
 
     <filter-mapping>
         <filter-name>${project.parent.artifactId}RequireLoginLocationFilter</filter-name>
-        <url-pattern>/*</url-pattern>
+        <url-pattern>/</url-pattern>
+        <url-pattern>*.page</url-pattern>
```

Comment:
Doesn't **/*** match both these?

Reply:
I think I see what you are doing, looks like you're technically "switching" the filter logic to instead be handled via the filter mapping pattern but also include the home page, right?

Reply:
You are right:  I switched the filter logic in the filter mapping as it's more logic :)  and include the home page.

The other solution would be to modify the method `skipFilter` to take in account the home page.

Reply:
Sounds good to me